### PR TITLE
HotFix - Update RedirectController and its tests

### PR DESCRIPTION
**Pull request by IINamelessII**

### CODE REVIEW

File: src/main/java/shortener/httphandler/RedirectController.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
       return HttpResponse.redirect(new URI(aliasRecord.url()));
     } catch (NotFound exc) {
-      return HttpResponse.notFound("Alias not found.");
+      return HttpResponse.notFound("Alias was not found!");
```
Comment (yugisu) [inline src/main/java/shortener/httphandler/RedirectController.java:40]: @IINamelessII  Could you please revert this change as the previous variant is more neutral and does not carry any emotional payload with itself?

**Review by AshenNagamaki — APPROVED** [review]

**Review by yugisu — CHANGES_REQUESTED** [review]

File: src/main/java/shortener/httphandler/RedirectController.java (hunk 1 above)
Comment (IINamelessII) [reply src/main/java/shortener/httphandler/RedirectController.java:40]: @yugisu Using "!" is a regular practice for errors' messages

Comment (yugisu) [reply src/main/java/shortener/httphandler/RedirectController.java:40]: @IINamelessII Understandable, have a nice day

**Review by yugisu — APPROVED** [review]

File: src/main/java/shortener/httphandler/RedirectController.java (hunk 1 above)
Comment (IINamelessII) [reply src/main/java/shortener/httphandler/RedirectController.java:40]: @yugisu Thanks for understanding
